Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -140,10 +140,10 @@ public string ToolTipText
         {
             set
             {
-                if (value != toolTipText)
```

Comment:
>if (value != toolTipText) [](start = 16, length = 25)

Given that this was removed - was it intended that setting the text to the same thing should also trigger a local content refresh (even if it's being set to the same thing?)

Reply:
In editor mode yes, because you can't count on Update to be called consistently.